src/manuel/tests/test_module.py

```diff
@@ -4,6 +4,7 @@
 import manuel
```

Comment:
What is the motivation for moving this file?

Reply:
I removed the separate test directiory and went back to the former flat structure.